db/migrate/20171116140737_create_client_notifications.rb

```diff
@@ -0,0 +1,12 @@
+class CreateClientNotifications < ActiveRecord::Migration[5.1]
+  def change
+    create_table :client_notifications do |t|
+      t.text :body
+      t.references :user, foreign_key: true
+      t.boolean :active
+      t.datetime :deleted_at
```

Comment:
Just for your information, no need to change this:
```ruby
t.datetime :deleted_at, index: true
```

this way the add_index at the bottom is not needed.

Reply:
👍